package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mineflayer-item-map-downloader",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
I will make the version change in a release commit. Please don't change the version in the package.json